app/controllers/dashboard/contacts_controller.rb

```diff
@@ -2,7 +2,7 @@ class Dashboard::ContactsController < Dashboard::BaseController
   before_action :set_contact, only: [:show, :edit, :update, :destroy]
 
   def index
-    @contacts = current_account.contacts
+    @contacts = current_account.contacts.page(params[:page]).per(10)
```

Comment:
We should have consistency in global configuration across models.

https://github.com/kaminari/kaminari#configuring-kaminari

Seem 25 by default is good, but if you want to customize into 20 is fine with me, but put it in global config.